docs/1-tendermint-overview.md

```diff
@@ -0,0 +1,92 @@
+# Tendermint protocol overview
+
+We are going to present a very short overview of the [Tendermint](https://github.com/tendermint) consensus algorithm and the main aspects that are necessary in order to understand the fork accountability algorithm. 
+Please note that this is note an exhaustive and formal description, we invite the reader to check out the [Tendermint documentation](https://docs.tendermint.com/master/) and the [official paper](https://arxiv.org/abs/1807.04938) about the consensus algorithm.   
+
+## Consensus algorithm
+
+Tendermint is a partially synchronous BFT consensus protocol. The protocol requires a fixed set of validators that attempt to come to consensus on one value or, more precisely, on a block (a block is a list of transactions) using gossip-based communication. The consensus algorithm proceeds in rounds: in each round there's a validator that proposes a value (proposer) and all the validators then vote during the round on whether to accept the proposed value or move on to the next round (proposers are chosen according to their voting power).
+
+As most of the consensus algorithms, the Tendermint consensus algorithm assumes a correct majority of processes: less than 1/3 of processes are faulty (*f*), therefore the total number of processes *n* is strictly greater than *3f* (*n > 3f*). 
+
+During a round, processes can exchange different messages and we can summarize the execution in two voting steps: Prevote and Precommit steps. A vote can be for a particular value or for a null value.
+
+A correct process decides on a value *v* upon receiving a proposal and *2f + 1* Precommit messages for *v* in a round *r*. A correct process sends a Precommit message for a value *v* upon receiving a proposal and *2f + 1* Prevote messages for *v* in a round *r*.
+
+Validators wait some time before sending a Prevote with a null value if they don't receive a valid proposal after a certain time and they send a Precommit message with a null value if they don't receive *2f + 1* Prevote messages for a value.
+If a correct process receives more than 2/3 Precommit messages for a null value in the same round, it moves to the next round.
+
+After a decision has been made, processes continue to agree on other values on another height (consensus instance) and they repeat the process described above in order to agree on different transactions.
+
+In order to ensure that processes will eventually come to a decision, there are some constraints that are applied. These rules are used to prevent any malicious attempt to cause more than one block to be committed at a given height. 
+When a validator sends a Precommit for a value *v* at round *r*, we say it is *locked* on *v*. Validators can propose and send Prevote messages for a value they have locked and they can only change the locked value if they receive a more recent proposal with a quorum of Prevote messages. 
```

Comment:
Maybe put "validator" instead of "it" in the first sentence. Maybe someone won't understand who "it" is.

Reply:
Fixed